src/components/request/specifyRequest.js

```diff
@@ -68,6 +68,7 @@ class SpecifyRequest extends React.Component {
             value={this.state.amount.value}
             onChange={this.handleChange.bind(this, 'amount')}
             t={this.props.t}
+            isRequesting
```

Comment:
can't find the value of `isRequesting` in any line along `specifyRequest.js` can you clarify where is coming from or how is initialised?

Reply:
This is short for `isRequesting = true` so it works like switch between Requesting and sending module

Reply:
all clear 🌮